app/scripts/proactive/view/TaskView.js

```diff
@@ -37,7 +37,11 @@ define(
             var iconPath = this.icons[this.modelType];
             try{
 	            iconPath = this.iconsPerLanguage[this.model.get("Execute").get("Script").get("Language")];
-            }catch(err){}
+            }catch(err){
+            	 try{
+     	            iconPath = this.iconsPerLanguage[this.model.get("Execute").Script.Language];
```

Comment:
it is possible to access json attributes in javascript either by their attribute key as a string, or directly accessing it with the dot notation.